package.json

```diff
@@ -64,6 +64,7 @@
     "build:patch:local": "cross-env UPGRADE_TYPE=patch PUBLISH_ENDPOINT=local USE_IN_MEMORY_REGISTRY=1 ./bootstrap/scripts/buildPatch.sh",
     "build:patch:local:ci": "cross-env UPGRADE_TYPE=patch PUBLISH_ENDPOINT=local BUILD_ENV=ci USE_IN_MEMORY_REGISTRY=1 ./bootstrap/scripts/buildPatch.sh",
     "build:patch:local:ci:nightly": "./bootstrap/scripts/buildNightly.sh",
+    "build:patch:local:ci:nightly:noparam": "PARAMS=none ./bootstrap/scripts/buildNightly.sh",
```

Comment:
maybe add 'cross-env' in front of the params similar to the other scripts above? 

`cross-env PARAMS=none ./bootstrap/scripts/buildNightly.sh`